test/DynamoCoreTests/Performance/PerformanceTests.cs

```diff
@@ -85,6 +108,12 @@ private void DoWorkspaceOpenAndCompare(string filePath)
             var wcd2 = new serializationTestUtils.WorkspaceComparisonData(ws2, CurrentDynamoModel.EngineController, dsExecution: false);
 
             serializationTestUtils.CompareWorkspaceModelsMSIL(wcd1, wcd2);
+
+            var newEngineExecutionTime = model.EngineController.ExecutionTime;
+
+            Console.WriteLine("Execution time old Engine={0} ms, new Engine={1} ms", oldEngineExecutionTime, newEngineExecutionTime);
```

Comment:
does this definitely make it into the test log?

I don't think this is going to give us very stable metrics when run with the parallel test job - this job will run in serial, but other threads on the machine will be running other tests...

I guess we'll have to run this these tests either with the master-15 self serve, locally, or create another test job.

Reply:
I will check if this goes into the log now. Yes, I was more thinking about using these numbers locally now.

Reply:
Yes, the numbers do show up in the log. I added an example to the PR.